src/main/java/mate/academy/spring/dto/request/MovieSessionRequestDto.java

```diff
@@ -1,10 +1,15 @@
 package mate.academy.spring.dto.request;
 
 import java.time.LocalDateTime;
+import javax.validation.constraints.FutureOrPresent;
+import javax.validation.constraints.Positive;
 
 public class MovieSessionRequestDto {
+    @Positive
     private Long movieId;
```

Comment:
https://javaee.github.io/javaee-spec/javadocs/javax/validation/constraints/Positive.html#:~:text=null%20elements%20are%20considered%20valid.

Long should not be null
so let's add javax.validation.constraints.NotNull everywhere above Long as well